<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLBeamParametersTabWidget</class>
 <widget class="QTabWidget" name="qMRMLBeamParametersTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tabGeometry">
   <attribute name="title">
    <string>Geometry</string>
   </attribute>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_SAD">
      <property name="toolTip">
       <string>Source axis distance (SAD)</string>
      </property>
      <property name="text">
       <string>Source distance (mm):</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_SAD">
      <property name="toolTip">
       <string>Source axis distance (SAD)</string>
      </property>
      <property name="minimum">
       <double>50.000000000000000</double>
      </property>
      <property name="maximum">
       <double>5000.000000000000000</double>
      </property>
      <property name="value">
       <double>2000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_MLCBoundaryAndPosition">
      <property name="text">
       <string>MLC positions:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_MLCBoundaryAndPositionTable">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="nodeTypes">
       <stringlist>
        <string>vtkMRMLTableNode</string>
       </stringlist>
      </property>
      <property name="noneEnabled">
       <bool>true</bool>
      </property>
      <property name="addEnabled">
       <bool>false</bool>
      </property>
      <property name="removeEnabled">
       <bool>false</bool>
      </property>
      <property name="selectNodeUponCreation">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_XjawPosition">
      <property name="text">
       <string>X jaw positions (mm):</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="ctkRangeWidget" name="RangeWidget_XJawsPosition">
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="minimum">
       <double>-200.000000000000000</double>
      </property>
      <property name="maximum">
       <double>200.000000000000000</double>
      </property>
      <property name="minimumValue">
       <double>-100.000000000000000</double>
      </property>
      <property name="maximumValue">
       <double>100.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_YJawPosition">
      <property name="text">
       <string>Y jaw positions (mm):</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="ctkRangeWidget" name="RangeWidget_YJawsPosition">
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="minimum">
       <double>-200.000000000000000</double>
      </property>
      <property name="maximum">
       <double>200.000000000000000</double>
      </property>
      <property name="minimumValue">
       <double>-100.000000000000000</double>
      </property>
      <property name="maximumValue">
       <double>100.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_GantryAngle">
      <property name="text">
       <string>Gantry angle (°):</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="ctkSliderWidget" name="SliderWidget_GantryAngle">
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="maximum">
       <double>359.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="label_CollimatorAngle">
      <property name="text">
       <string>Collimator angle (°):</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="ctkSliderWidget" name="SliderWidget_CollimatorAngle">
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="maximum">
       <double>359.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label_CouchAngle">
      <property name="text">
       <string>Couch angle (°):</string>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="ctkSliderWidget" name="SliderWidget_CouchAngle">
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="maximum">
       <double>359.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_VSADx">
      <property name="text">
       <string>Source distance X (mm):</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_VSADy">
      <property name="text">
       <string>Source distance Y (mm):</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_VSADx">
      <property name="maximum">
       <double>30000.000000000000000</double>
      </property>
      <property name="value">
       <double>5400.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_VSADy">
      <property name="maximum">
       <double>30000.000000000000000</double>
      </property>
      <property name="value">
       <double>4500.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_ScanSpotParameters">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Scan spot parameters:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_ScanSpotParametersTable"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_DistanceMLC">
      <property name="text">
       <string>Distance MLC (mm):</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_DistanceMLC">
      <property name="maximum">
       <double>30000.000000000000000</double>
      </property>
      <property name="value">
       <double>400.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_Isocenter">
      <property name="text">
       <string>Isocenter:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="ctkCoordinatesWidget" name="CoordinatesWidget_Isocenter">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="16" column="0" colspan="2">
     <widget class="ctkCollapsibleButton" name="CollapsibleButton_RangeShifterParameters">
      <property name="text">
       <string>Range shifter parameters</string>
      </property>
      <property name="collapsed">
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Isocenter to range shifter distance:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="Label_IsocenterToRangeShifterDistance">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Setting:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="Label_RangeShifterSetting">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="Label_RangeShifterType">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_1">
           <property name="text">
            <string>ID:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Type:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="Label_RangeShifterID">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>WET:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="Label_RangeShifterWET">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="15" column="0" colspan="2">
     <widget class="ctkCollapsibleButton" name="CollapsibleButton_SnoutParameters">
      <property name="text">
       <string>Snout parameters</string>
      </property>
      <property name="collapsed">
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>ID</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="Label_SnoutID">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Position:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="Label_SnoutPosition">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tabVisualization">
   <attribute name="title">
    <string>Visualization</string>
   </attribute>
   <layout class="QFormLayout" name="formLayout_3">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="beamSEyeViewLabel">
      <property name="text">
       <string>Beam's eye view</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="checkBox_BeamsEyeView"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="includeContourInBEWLabel">
      <property name="text">
       <string>Include contour in BEW</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="checkBox_ContoursInBEW"/>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_UpdateDRR">
      <property name="text">
       <string>Update DRR</string>
      </property>
      <property name="icon">
       <iconset resource="../../../../DicomRtImportExport/SubjectHierarchyPlugins/Resources/qSlicerDicomRtImportExportSubjectHierarchyPlugins.qrc">
        <normaloff>:/Icons/PlanarImage.png</normaloff>:/Icons/PlanarImage.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tabMultiLeafCollimator">
   <attribute name="title">
    <string>Multi Leaf Collimator</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="ctkCollapsibleButton" name="CollapsibleButton_MLCBoundary">
      <property name="text">
       <string>Boundary parameters</string>
      </property>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_MLCType">
         <property name="text">
          <string>MLC type:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="radioButton_MLCX">
           <property name="text">
            <string>MLCX</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_MLCType</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_MLCY">
           <property name="text">
            <string>MLCY</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_MLCType</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_NumberOfLeafPairs">
         <property name="text">
          <string>Number of leaf pairs:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="ctkSliderWidget" name="SliderWidget_NumberOfLeafPairs">
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>300.000000000000000</double>
         </property>
         <property name="value">
          <double>16.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_LeafPairBoundarySize">
         <property name="text">
          <string>Leaf pair size (mm):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="ctkSliderWidget" name="SliderWidget_LeafPairBoundarySize">
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="value">
          <double>5.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_IsocenterOffset">
         <property name="text">
          <string>Isocenter offset (mm):</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="ctkSliderWidget" name="SliderWidget_IsocenterOffset">
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="pushButton_GenerateMLCBoundary">
         <property name="text">
          <string>Generate new MLC boundary table</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="pushButton_UpdateMLCBoundary">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Update selected MLC table</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="checkBox_ParallelBeam">
         <property name="text">
          <string>Parallel beam</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="ctkCollapsibleButton" name="CollapsibleButton_MLCPosition">
      <property name="text">
       <string>Position</string>
      </property>
      <layout class="QFormLayout" name="formLayout_5">
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_CalculateMLCPosition">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Calculate MLC position</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tabScanSpot">
   <attribute name="title">
    <string>Scan Spot Weight</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="qMRMLTableView" name="MRMLTableView_ScanSpotWeight">
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCoordinatesWidget</class>
   <extends>QWidget</extends>
   <header>ctkCoordinatesWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkRangeWidget</class>
   <extends>QWidget</extends>
   <header>ctkRangeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLTableView</class>
   <extends>QTableView</extends>
   <header>qMRMLTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../DicomRtImportExport/SubjectHierarchyPlugins/Resources/qSlicerDicomRtImportExportSubjectHierarchyPlugins.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>MRMLNodeComboBox_ScanSpotParametersTable</sender>
   <signal>currentNodeChanged(vtkMRMLNode*)</signal>
   <receiver>MRMLTableView_ScanSpotWeight</receiver>
   <slot>setMRMLTableNode(vtkMRMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup_MLCType"/>
 </buttongroups>
</ui>
